tests/JobDefinitionFactoryTest.php

```diff
@@ -131,17 +215,22 @@ public function testCreateJobDefinitionWithConfigIdAndBackend(): void
             'projectId' => 'my-project',
             'componentId' => 'my-component',
             'configId' => 'my-config',
-            'backend' => [
-                'type' => 'custom',
-            ],
+            'backend' => $backendData,
         ];
 
         $jobDefinitions = $this->createJobDefinitionsWithConfiguration($jobData, $configuration);
 
-        self::assertSame(
-            'custom',
-            $jobDefinitions[0]->getConfiguration()['runtime']['backend']['type'] ?? null
-        );
+        if ($expectedBackendData !== null) {
```

Comment:
No to je teda dost znásilněnej test, není lepší copy paste? anebo změnit co expectedBackend data tak, aby tam byly celý?